.github/ISSUE_TEMPLATE/user-story-template.md

```diff
@@ -25,6 +25,27 @@ Then
 - [ ] Checklist item
 - [ ] Meets the DOD
 
+**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)
+- [ ] User story is included
+- [ ] User role and type are identified
+- [ ] Acceptance criteria are included
+- [ ] Wireframes are included (if required)
+- [ ] Does this User Story need stakeholder approval?
+- [ ] Design / Solution is accepted by Product Owner
+- [ ] Dependencies are identified (technical, business, regulatory/policy)
+- [ ] Story has been estimated (under 13 spts)
+
+·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)
+- [ ] Passes developer unit and automated tests
```

Comment:
Passing the tests is an automated check from our CI pipeline. Whether we use unit or integration tests is also up for debate.
```suggestion
- [ ] Acceptance criteria are tested by the CI pipeline
```